tests/test_ind_indygo_gov_audit.py

```diff
@@ -31,37 +31,34 @@ def test_tests():
 
 
 def test_title():
-    assert parsed_items[0]["title"] == "IndyGo Governance Audit Committee"
+    assert parsed_items[0]["title"] == "IndyGo Governance and Audit Committee"
 
 
 def test_description():
     assert parsed_items[0]["description"] == ""
 
 
 def test_start():
-    assert parsed_items[0]["start"] == datetime(2023, 1, 19, 13, 0)
+    assert parsed_items[0]["start"] == datetime(2024, 1, 18, 13, 0)
 
 
 # def test_end():
 #     assert parsed_items[0]["end"] == datetime(2019, 1, 1, 0, 0)
 
 
 def test_time_notes():
-    assert (
-        parsed_items[0]["time_notes"]
-        == "Governance Audit Committee meetings are set for 1:00PM unless otherwise noted in meeting description. Please double check the website before the meeting date."  # noqa
-    )  # noqa
+    assert parsed_items[0]["time_notes"] == ""  # noqa  # noqa
```

Comment:
See this [comment](https://github.com/City-Bureau/city-scrapers-indianapolis/pull/14/files#r1463728936), re: suppressing linter issues.